project-studying-dotnet/Streetcode-Admin
# Add Fluent Validation to StreetCode

# Summary
- Add FluentValidation package to BLL and UnitTests projects
- Add FluentValidation.DependencyInjection package to WebApi project
- Create MediatR Validation PreProcessor
- Create interface markers for Validation PreProcessor

# Investigate link
- https://github.com/project-studying-dotnet/Streetcode-Admin/issues/101

# How to create validators
- Ensure that your command is implements `IValidatableRequest<TResponse>` interface
- Create class `<command name>Validator` near the command that you want to test
- Ensure that your Validator classs is inhereted from `AbstractValidator<T>`
- Implement your validation logic

# Closing
- close #84
